src/main.js

```diff
@@ -0,0 +1,32 @@
+var server = require('http').createServer();
+var io = require('socket.io')(server);
+import {Monopoly} from "./monopoly.js";
+import {PlayerList} from "./player.js";
+
+
+
+
+io.on('connect', function(socket){
+  //assign user
+  socket.on('getAccountVal', function(playerId){
```

Comment:
This approach is fine, but I'm still going to talk about the way I would've done it and why.  

Basically whenever a new turn starts, I would send the player the entire board state in one go.  So they would get a JSON string that looks like: 

```
{
    playerTurn: 'jim',
    gameState: {
        freeParkingCash: 7000,
    },
    playerState: {
        jim: {
            cashMoney: 582,
            cards: [3, 71, 12, 52], // basically some id corresponding with a card or somefin
            properties: [8, 12, 4],
            location: 42, // or 'jail', etc. 
            piece: 'topHat',
        },
        jake: {
            // Same stuff as above...
        },
    },
    propertyStates: [
        1: {
            isMortgaged: false,
            houses: 0, // 5 indicates hotel? 
            // whatever details
        },
        // etc. etc.
    ],
}
```

You could even put the full details for each property in propertyStates, and card details in cardStates.  That way you could even have different 'versions' of monopoly if you wanted with different sale prices/etc.  Or just so that the AIs can use it as a reference card.

So the reason I'd do this over requiring Remote Procedure Calls (RPC) like you're doing is that this is all the data AIs are going to want to request anyway.  So by just giving all of it to them at the start, you reduce the traffic to the server and I think it's a little simpler implementation.